public/index.html

```diff
@@ -168,103 +187,168 @@
-    <div class="instructions nodisplay">You must first <a target="_blank"
-    href="https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap">install
-    the Chrome extension</a> then come back to this page.
-
-    If it is already installed, please enable it and reload this page.
-    </div>
-    
-    <div class="network-config nodisplay">
-      Choose the network to connect:
-      <p>
-      <mwc-select id="networks" fixedMenuPosition label="Network">
-        <mwc-list-item selected value="https://main.agoric.net/network-config">Agoric Mainnet</mwc-list-item>
-        <mwc-list-item class="experimental nodisplay" value="https://main.agoric.net/network-config?coinType=118">Agoric Mainnet (118)</mwc-list-item>
-        <mwc-list-item value="https://testnet.agoric.net/network-config">Agoric Testnet</mwc-list-item>
-        <mwc-list-item class="experimental nodisplay" value="https://testnet.agoric.net/network-config?coinType=118">Agoric Testnet (118)</mwc-list-item>
-        <mwc-list-item value="https://devnet.agoric.net/network-config">Agoric Devnet</mwc-list-item>
-        <mwc-list-item value="https://stage.agoric.net/network-config">Agoric
-        Stage</mwc-list-item>
-        <mwc-list-item value="cosmoshub-testnet">Cosmos Hub Testnet</mwc-list-item>
-      </mwc-select>
-      <button id="chooseNetworkConfig">Configure</button>
-      </p>
-      <div id="keplr-status"></div>
-    </div>
-  </mwc-dialog>
+    <main class="container">
+      <div id="intro">
+        <h1>Agoric Wallets</h1>
+
+        <p>Choose the wallet that best suits your needs.</p>
+      </div>
+
+      <div id="dapp-clients">
+        <h4 class="upcase">Dapp Client Wallets</h4>
+
+        <p>These wallets support offer-safe interaction with Agoric dapps.</p>
+      </div>
+
+      <div id="ag-solo">
+        <h2>Ag-Solo</h2>
+
+        <p>Docker-based</p>
+      </div>
+
+      <a
+        class="button"
+        id="ag-solo-install"
+        href="https://agoric.com/documentation/getting-started/beta.html#set-up-a-wallet-agoric-solo-machine"
+        >Learn&nbsp;How</a
+      >
+
+      <div id="web-solo" class="experimental nodisplay">
+        <h2>Web-Solo</h2>
+
+        <p>Web Page</p>
+      </div>
+
+      <a
+        class="nodisplay button experimental"
+        id="web-solo-install"
+        href="web-solo/current/"
+        >Try&nbsp;It</a
+      >
+
+      <div class="border-right"></div>
+
+      <div id="cosmos-clients">
+        <h4 class="upcase">Cosmos-Layer Wallets</h4>
+
+        <p>
+          These wallets enable staking and trading chain tokens including RUN
+          and BLD.
+        </p>
+
+        <p>
+          Unfortunately, you can't use them to interact with Agoric dapps; for
+          that you must use a Dapp Client wallet.
+        </p>
+      </div>
+
+      <div id="keplr">
+        <h2>Keplr</h2>
+        <p>
+          Chrome Extension
+          <a
```

Comment:
Sorry about the diff from code formatting. This is the only change here.

Reply:
I'm not in a big rush. I'd rather you undo the formatting changes.

Or at least: put them in their own commit.

Reply:
Is the resulting format standardized at all? Is it what vs-code produces? Where did the previous formatting come from?

Reply:
It was from the prettier plugin in vs-code. I just removed the formatting changes from this PR though. I think we probably should be using it though.

Reply:
> It was from the prettier plugin in vs-code. I just removed the formatting changes from this PR though. I think we probably should be using it though.

Rather than requiring IDE plugins, I'd prefer to set up a `"lint"` and `"lint:prettier"` script in `package.json`.  Then there is something people can run without needing a particular plugin (though of course, given that, you could still use the plugin if you wanted).

Reply:
SGTM, I'll do a follow-up PR for that